Tools/scripts/summarize_stats.py

```diff
@@ -313,7 +314,7 @@ def kind_to_text(kind: int, opcode: str):
     def is_specializable(self, opcode: str) -> bool:
         return "specializable" in self._get_stats_for_opcode(opcode)
 
-    def get_specialized_total_counts(self) -> tuple[int, int, int]:
+    def get_specialized_total_counts(self) -> tuple[int, int, int, int]:
```

Comment:
Again, keep unrelated changes out of the PR. Please make a new PR for fixing the type hints, and revert these.